.github/workflows/docs.yaml

```diff
@@ -1,5 +1,4 @@
 name: docs-ci
-permissions: {}
```

Comment:
So this will end up having
```yaml
permissions:
  contents: write
```
from below.

Do we want it like that or do we want empty permissions here?

Reply:
We want `write` permissions not the empty one.